app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert split_integer(10, 2) == [5, 5]
```

Comment:
this test is the same as the second one. Instead, here you should check that sum of parts equals the initial value